src/docs/getting-started/templates/README.md

```diff
@@ -24,8 +24,11 @@ dotnet new -i OrchardCore.ProjectTemplates::1.3.0-* --nuget-source https://nuget
 
 #### Generate an Orchard Cms Web Application
 
+!!! warning
+    Due to a bug in the current published version, the following `dotnet new` commands will require the extra argument `--orchard-version 1.3.0`. For instance, instead of typing `dotnet new occms` use `dotnet new occms --orchard-version 1.3.0`
```

Comment:
Add a note about removing this to the release publishing docs, so next time we'll remove both that and this warning.

Reply:
done